types/storybook__react/index.d.ts

```diff
@@ -16,7 +16,7 @@ export type RenderFunction = () => Renderable | Renderable[];
 export interface DecoratorParameters {
     [key: string]: any;
 }
-export type StoryDecorator = (story: RenderFunction, context: { kind: string, story: string }) => Renderable | null;
+export type StoryDecorator = (story: RenderFunction, context: { kind: string; story: string }) => Renderable | null;
```

Comment:
this is a simple object, not an interface, I think we'd keep a comma for these?

Reply:
More auto formatting that I can remove.

Reply:
Actually this appears to be happening from a husky pre-commit hook that's part of the DefinitelyTyped package.json file.

<img width="537" alt="Screen Shot 2019-07-09 at 10 29 03 AM" src="https://user-images.githubusercontent.com/3212954/60901942-6a7bad00-a234-11e9-8948-de56f9bb80d0.png">